aiida_common_workflows/common/init_magnetization.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to manage initial magnetization.
+"""
+from aiida.orm import StructureData
+
+
+class InitMagnetizationManager():
+    """
+    A manager to facilitate the management of initial magnetizations.
+    The API agreed in the project impose to pass the initial magnetizations
+    as a list with the spin polarization (difference between spin up and down electrons)
+    for each site.
+    The manager requires in input for the initaialization this list and the structure
+    and contains methods to manage these quantities.
+    """
+
+    def __init__(self, structure, initial_spin_site):
```

Comment:
Why `initial_spin_site` and not `initial_magnetization` just as for the builder?

Reply:
Ok

Reply:
I guess it was to remind it is a list of the spin per site? `initital_magnetization` seems to be too general.

Reply:
I guess we could rename it to `magnetization_per_site`, this way it is explicit and I don't think the `initial` adds anything. That is implied. It is not going to ever be the magnetization that the code is going to set halfway through

Reply:
Perfect. Agreed on `magnetization_per_site`.